app/models/threads.js

```diff
@@ -307,13 +307,47 @@
             }
             if (updates.distribution && 'expression' in updates.distribution &&
                 updates.distribution.expression != this.get('distribution')) {
-                const dist = updates.distribution.expression;
+               const dist = updates.distribution.expression;
+                const oldDist = this.get('distribution');
+
                 const normalized = await F.atlas.resolveTagsFromCache(dist);
+                const oldNormalized = await F.atlas.resolveTagsFromCache(oldDist);
+                
                 if (normalized.universal !== updates.distribution.expression) {
                     F.util.reportError('Non-universal expression sent by peer',
                                        {distribution: dist});
                 }
-                this.addNotice("Distribution Changed", normalized.pretty);
+                
+                var removed = null;
```

Comment:
Use `let` or `const` instead of `var` and no need to initialize to null, the default init value is `undefined` which is falsy enough.